extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -101,8 +106,7 @@ export class AmpAnim extends AMP.BaseElement {
   /** @override */
   unlayoutCallback() {
     // Release memory held by the image - animations are typically large.
-    this.img_.src = 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///' +
-        'yH5BAEAAAAALAAAAAABAAEAAAIBRAA7';
+    this.img_.src = SRC_PLACEHOLDER;
```

Comment:
I updated this so that we update the `srcset` as well. But empirically (Chrome/Firefox/Safari Desktop MacOS), this override will actually update the actual image src when we scroll outside the viewport, even if srcset attribute is present. I'm not quite sure why.